testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/InternationalizationTest.java

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
Tests ignored as suggested by @ssilvert on this [comment](https://github.com/keycloak/keycloak/issues/16515#issuecomment-1397134888).